blocks/index.js

```diff
@@ -17,5 +17,6 @@ export { default as AlignmentToolbar } from './alignment-toolbar';
 export { default as BlockControls } from './block-controls';
 export { default as BlockDescription } from './block-description';
 export { default as Editable } from './editable';
+export { default as EditableProvider } from './editable/provider';
```

Comment:
The minimum requirement is that the editor provides the context assumed to exist by the `Editable` component. This could be implemented within `editor` and not exposed, but then the coupling between `Editable` and its context becomes unclear.

Personally I think it should be left here. If we want to discourage particular exports, maybe we could do so merely by omitting documentation